### PR TITLE
Make public hintText field of the InputDecoration of message_input

### PR DESCRIPTION
**Pull request by TramPamPam**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Comment by TramPamPam** [comment]
@imtoori @xsahil03x @tbarbugli

Could someone review this pull request, please?

File: lib/src/message_input.dart [hunk 1]
```diff
@@ -151,6 +151,9 @@ class MessageInput extends StatefulWidget {
   /// Color used for attachment icon.
   final Color attachmentIconColor;
 
+  /// Text for the hint of InputDecoration. Default is 'Write a message'.
+  final String hintText = 'Write a message';
```
Comment (imtoori) [inline lib/src/message_input.dart:155]: I think we should add this property in the widget constructor

**Review by imtoori — CHANGES_REQUESTED** [review]
I added a comment
Also, we could move it to the theme. I think it would be better because the MessageInput is also used while editing a message and to customise it you should set `editMessageInputBuilder` and that's not so nice.

**Comment by TramPamPam** [comment]
@imtoori I've added this property to widget constructor. Concerning theme - I'm not sure where to add this field.

**Comment by TramPamPam** [comment]
@imtoori any update on this?

**Comment by imtoori** [comment]
Hey @TramPamPam I'll add it in the theme and I'll ping you once it's done

**Comment by imtoori** [comment]
Closing this in favour of #172 
@TramPamPam let me know what do you think about it

**Comment by TramPamPam** [comment]
@imtoori 
That should do. Thanks!

**Pull request closed without merging**